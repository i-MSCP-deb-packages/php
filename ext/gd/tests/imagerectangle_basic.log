
---- EXPECTED OUTPUT
282aaecfdd50091821d63093d9bb1bb9
---- ACTUAL OUTPUT
201371a869780961f0c4c9c9634de90b
---- FAILED
