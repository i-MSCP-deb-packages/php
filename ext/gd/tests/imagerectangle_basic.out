201371a869780961f0c4c9c9634de90b